DigitalOffice/Services/ProjectService/Startup.cs

```diff
@@ -19,20 +29,34 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddHealthChecks();
-
             services.AddDbContext<ProjectServiceDbContext>(options =>
             {
                 options.UseSqlServer(Configuration.GetConnectionString("SQLConnectionString"));
             });
 
             services.AddControllers();
+
+            CreateTransientObjects(services);
+        }
+
+        private void CreateTransientObjects(IServiceCollection services)
```

Comment:
Create separate methods for each type of objects e.g. repositories, mappers ....

Reply:
Fixed